cobigen-eclipse/cobigen-eclipse/.classpath

```diff
@@ -20,7 +20,7 @@
 	<classpathentry exported="true" kind="lib" path="lib/mmm-util-core-7.4.0.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/slf4j-api-1.7.21.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/javaplugin-model-2.0.0.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/core-5.2.0.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/core-api-5.2.0.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/core-5.3.0-SNAPSHOT.jar" sourcepath="/core"/>
+	<classpathentry exported="true" kind="lib" path="lib/core-api-5.3.0-SNAPSHOT.jar" sourcepath="/core-api"/>
```

Comment:
Is this `sourcepath="/core-api` needed? I would remove it if not.

Reply:
It does. For example the [FileInputConverter](https://github.com/devonfw/tools-cobigen/blob/master/cobigen-eclipse/cobigen-eclipse/src/com/devonfw/cobigen/eclipse/generator/generic/FileInputConverter.java) imports some exceptions from the core-api.

Reply:
Okay, got it. Thanks :)

Reply:
sourcePath does not have anything to do with imports. It's just an eclipse marker for source lookup in eclipse. So if you navigate on the classpath to a specific class of a dependency, you can specify that your sources are located in a workspace project like /core.
As we are using the same setup for our workspaces, it is anyhow save to be commited. Might make life easier for others in case someone debuggs over library boarders.